modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -87,17 +87,17 @@ Use the following values to configure the X12 GS identifier type:
 [[reference-id]]
 == Outbound Message Flow Partner Reference Identifiers
 
-Outbound message flows that transform your backend application messages to EDI format use the partner Reference ID when sending data to your partners.
+Outbound message flows that transform your backend-application messages to EDI format use the partner Reference ID when sending data to your partners.
```

Comment:
Just want to doublecheck that you do mean backend-application, not back-end application.